### PR TITLE
Events Socket Emit

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
Create a socket emit for Events model

### CODE REVIEW

**Comment by iamigo** [comment]
@tausifmuzaffar test failures?

**Comment by kamilsmuga** [comment]
tests fixed @pallavi2209 @pallavisingh9 @iamigo

File: db/model/event.js [hunk 1]
```diff
@@ -68,6 +81,25 @@ module.exports = function event(seq, dataTypes) {
...
+      afterCreate: (instance) => {
+        const changedKeys = Object.keys(instance._changed);
+        const ignoreAttributes = ['isDeleted'];
+        return realTime.publishObject(instance.toJSON(),
+          botEventNames.add, changedKeys, ignoreAttributes, pubOpts);
+      },
+
+      afterUpdate(instance /* , opts */) {
+        return realTime.publishObject(instance.toJSON(),
+          botEventNames.upd, null, null, pubOpts);
```
Comment (iamigo) [inline db/model/event.js]: curious why not sending changed keys and ignore attributes here or in afterDelete, like you do in afterCreate?

File: tests/realtime/utils.js [hunk 2]
```diff
@@ -185,8 +192,21 @@ module.exports = {
     return JSON.parse(JSON.stringify(standardBotAction));
   },
 
+  getStandardEvent() {
+    return JSON.parse(JSON.stringify(standardEvent));
+  },
+
   forceDelete(done) {
-    tu.forceDelete(tu.db.Perspective, testStartTime)
+    tu.db.Event.destroy({
```
Comment (iamigo) [inline tests/realtime/utils.js]: curious why calling destroy directly rather than using tu.forceDelete?

Comment (tausifmuzaffar) [reply tests/realtime/utils.js]: tu.forcedelete uses name and the event doesn't have so need to use a destroy

File: db/model/event.js (hunk 1 above)
Comment (tausifmuzaffar) [reply db/model/event.js]: overlooked this; ive added the changes

**Comment by tausifmuzaffar** [comment]
@iamigo @kamilsmuga same issue about merging for #698

**Pull request merged**